tests/mesh/boundary_info.C

```diff
@@ -260,7 +260,7 @@ public:
       bi.nodeset_name(0) = "ZERO";
       bi.nodeset_name(1) = "ONE";
 
-      BoundaryInfo bi2 {bi};
+      BoundaryInfo & bi2 (bi);
```

Comment:
Yuck, how did I miss this before???  This commit isn't disambiguating a copy from a move, it's turning a copy into a no-op reference initialization!  That basically eliminates this unit test!